components/AssetsList/BaseGovernanceForm.tsx

```diff
@@ -228,5 +241,20 @@ const BaseGovernanceForm = ({ formErrors, form, setForm, setFormErrors }) => {
     </>
   )
 }
+const BaseGovernanceForm = ({
```

Comment:
Since the forms are pretty different at this point I just made a new component, but they have a unified API which just checks the `_programVersion` to decide which to display.

Reply:
I didn't end up reusing much code from the v2 form, but if I were, I think that composition is usually better than having a ton of `if` blocks in a component depending on the version.